### ISSUE TITLE
Remove the deprecated gtkreactor

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8145]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8145|
|Type|enhancement|
|Created|2015-12-18 12:07:53Z|
|Branch|https://github.com/twisted/twisted/tree/gtk-remove-8145|

The GTK 1 reactor has been deprecated since 2010 -- GTK1 is dead, and we can safely remove it.

<details><summary>Searchable metadata</summary>

```
trac-id__8145 8145
type__enhancement enhancement
reporter__hawkowl hawkowl
priority__normal normal
milestone__None None
branch__branches_gtk_remove_8145 branches/gtk-remove-8145
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1450440473059057 1450440473059057
changetime__1450775250897787 1450775250897787
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_1"></a>|@hawkowl commented|
|-|-|

(In [46524]) Branching to gtk-remove-8145.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

Builders spun, all green. Please review.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @hawkowl|
|-|-|

All good. Thanks! Please merge.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl set status to `closed`|
|-|-|

(In [46532]) Merge gtk-remove-8145: Remove the deprecated gtkreactor

Author: hawkowl
Reviewer: adiroiban
Fixes: [#8145](https://github.com/twisted/twisted/issues/8145)
